app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,5 @@
+class SessionsController < ApplicationController
```

Comment:
I think this should actually be removed for this pull request as its not needed until user story 13

Reply:
Done!